.github/workflows/validate-new-issue.yml

```diff
@@ -48,8 +56,5 @@ jobs:
 
         # Might get `gh issue comment` some day - https://github.com/cli/cli/issues/517
         echo -n "Commented: "
-        gh api "https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments" \
-          --method POST \
-          --input comment \
-          | jq .html_url
+        gh issue comment ${{ github.event.issue.number }} --body "$(cat comment)"
```

Comment:
nit: Would have been better to separate this into its own PR.

Reply:
Done: https://github.com/getsentry/.github/pull/34.